integrations/DCMFloodlight/browser.js

```diff
@@ -0,0 +1,300 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable class-methods-use-this */
+import get from "get-value";
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+import {
+  isDefinedAndNotNull,
+  isNotEmpty,
+  removeUndefinedAndNullValues,
+} from "../utils/commonUtils";
+import { NAME } from "./constants";
+import { transformCustomVariable, isValidFlag } from "./utils";
+
+class DCMFloodlight {
+  constructor(config, analytics) {
+    this.analytics = analytics;
+    this.advertiserId = config.advertiserId;
+    this.activityTag = config.activityTag;
+    this.groupTag = config.groupTag;
+    this.conversionEvents = config.conversionEvents;
+    this.conversionLinker = config.conversionLinker;
+    this.allowAdPersonalizationSignals = config.allowAdPersonalizationSignals;
+    this.doubleclickId = config.doubleclickId;
+    this.googleNetworkId = config.googleNetworkId;
+    this.name = NAME;
+  }
+
+  /**
+   * Ref - https://support.google.com/campaignmanager/answer/7554821
+   */
+  init() {
+    logger.debug("===In init DCMFloodlight===");
+
+    const sourceUrl = `https://www.googletagmanager.com/gtag/js?id=DC-${this.advertiserId}`;
+    ScriptLoader("DCMFloodlight-integration", sourceUrl);
```

Comment:
For other google products if `gtag` is already loaded won't this be redundant? Should we put a check in UI?

Reply:
@utsabc Not sure if the id is something that differs from product to product. For example, in the case of Google Optimize, the id part consists of the tracking id, and this is advertiser id.